data/json/obsoletion/obsolete-0.G-items.json

```diff
@@ -262,5 +262,51 @@
         "encumbrance": 20
       }
     ]
+  },
+  {
+    "id": "bio_power_armor_interface",
+    "copy-from": "bionic_general",
+    "type": "BIONIC_ITEM",
+    "name": { "str": "Power Armor Interface CBM" },
+    "looks_like": "bio_int_enhancer",
+    "description": "Interfaces the user's bionic power system with the internal charging port on suits of power armor, allowing them to draw from the user's bionic power banks.",
+    "price": 120000,
+    "price_postapoc": 500,
+    "weight": "500 g",
+    "difficulty": 1,
+    "extend": { "flags": [ "OBSOLETE" ] }
+  },
+  {
+    "id": "bio_power_armor_interface_mkII",
+    "copy-from": "bionic_general",
+    "type": "BIONIC_ITEM",
+    "name": { "str": "Power Armor Interface Mk. II CBM" },
+    "looks_like": "bio_int_enhancer",
+    "description": "Interfaces the user's bionic power system with the internal charging port on suits of power armor, allowing them to draw from the user's bionic power banks.  Twice as efficient as the Mk. I model.  The Mk. II was designed by DoubleTech Inc., to meet the popularity of the Mk. II power armor series.",
+    "price": 1000000,
+    "price_postapoc": 500,
+    "weight": "400 g",
+    "difficulty": 8,
+    "extend": { "flags": [ "OBSOLETE" ] }
+  },
+  {
+    "id": "bio_power_armor_interface",
```

Comment:
Same with actual bionic definitions - moving them around in files won't do much as characters will still have them as part of their character save json and at some point you'll have to delete them; you need to delete these entries and make a `bionic_migration` entry as described in the doc linked above to similar themed bionic or just nothing  to make them delete when loaded, similar to how it's done in https://github.com/CleverRaven/Cataclysm-DDA/blob/master/data/json/obsoletion/migration_bionics.json